### DISCUSSION TITLE
`supabase link` usage with SSL Enforcement Enabled

### DISCUSSION THREAD

**Discussion by r-southworth** [supabase/supabase#21058] [Questions]
When I attempt to link a project that has SSL enforcement enabled, I receive server error (FATAL: SSL required (SQLSTATE 08P01)). How do I utilize the certificate (downloaded from the dashboard) with the supabase CLI?

There are at least 3 posts on the discord about this, but no one seems to have any advice on the issue. 

https://discord.com/channels/839993398554656828/1202694226744180858

https://discord.com/channels/839993398554656828/1158511808089423973/1167147911616942201

https://discord.com/channels/839993398554656828/1123250671186100314

**Comment by marcusarnfast** [comment]
Im also having problems with this

**Comment by heavens-www** [comment]
I'm also having issues with the SSL when connecting with the pooler: FATAL: SSL connection is required (SQLSTATE XX000). I'm trying to use "supabase link".

**Comment by peterkos** [comment]
I was able to resolve this by updating my supabase vesrion `1.187.3 -> 1.187.8`. The error went away!

It showed warnings that I wasn't on the latest, but `1.187.8` isn't on the latest release tap yet. Installing via `supabase-beta` (instructions in the [readme](https://github.com/supabase/cli)) worked.
